experimental/templates/multiplatform-template/androidApp/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+plugins {
+    kotlin("multiplatform")
+    id("com.android.application")
+    id("org.jetbrains.compose")
+}
+
+kotlin {
+    android()
+    sourceSets {
+        val androidMain by getting {
+            dependencies {
+                implementation(project(":shared"))
+                implementation("androidx.appcompat:appcompat:1.5.1")
```

Comment:
We have this in `shared` module. No need to depend on it twice.

Reply:
Removed from shared (it is not needed actually in a shared module as Android specific is better to declare in androidApp)